single-node/user-data

```diff
@@ -5,7 +5,10 @@ set -e
 export ETCD_ENDPOINTS="http://127.0.0.1:2379"
 
 # Specify the version (vX.Y.Z) of Kubernetes assets to deploy
-export K8S_VER=v1.2.2_coreos.0
+export K8S_VER=v1.2.3_coreos.0-cni
+
+# Hyperkube image repository to use.
+export HYPERKUBE_IMAGE_REPO=quay.io/calico/hyperkube
```

Comment:
I also would still like enabling the calico image/services to be a feature flag given we are using custom images. In the single-node vagrant - I'm not opposed to having it default to "true", but given that it didn't seem super complex from the multi-node install to gate on a flag - it would be helpful during debugging to be able to flip this on/off.